antiplag/serializers.py

```diff
@@ -56,7 +56,7 @@ class DocumentDetailedSerializer(NonNullModelSerializer, serializers.ModelSerial
 
     class Meta:
         model = Document
-        fields = ("id", "name", "matches", "percentage")
+        fields = ("id", "name", "matches", "percentage", "intervals")
```

Comment:
I see the point of intervals variable now, but it still returns `None`

Was it part of task to update the endpoints?

Does it skip the `intervals` variable, when you want to deserialize data?

Reply:
it assures that no intervals get returned from api/submissions/{id} but gets returned for matched_docs from api/documents/{id}